lld/COFF/Writer.cpp

```diff
@@ -77,6 +77,12 @@ static unsigned char dosProgram[] = {
 static_assert(sizeof(dosProgram) % 8 == 0,
               "DOSProgram size must be multiple of 8");
 
+static char ltcg[] = "LTCG";
```

Comment:
`constexpr char ...[]`

constexpr/const make this internal linkage, making static unneeded